src/main/containers/Panels.jsx

```diff
@@ -6,12 +6,31 @@ import DestinationPanel from './DestinationPanel.jsx';
 const Panels = () => {
   const [activePanel, setActivePanel] = useState('source');
   const [dataTreeCount, setDataTreeCount] = useState(0);
+  const [data, setData] = useState(undefined);
+  // Tests are objects with 
+  // { payload: JSON that represents test,
+  //   status: initial value of '' };
+  const [tests, setTests] = useState(
+    [ {payload : JSON.stringify({ message: 'hello by joe', created_by: 'joe' }),
```

Comment:
lets remove the space between the array and curly brace. lets make all the objects in this array look similar to this
  [{
      payload: JSON.stringify({ message: 'hello by joe', created_by: 'joe' }),
      status: '',
    },

Reply:
I fixed this manually, was blocking other features